python/pyarrow/tests/parquet/test_parquet_file.py

```diff
@@ -329,3 +334,26 @@ def test_parquet_file_explicitly_closed(tempdir):
         p.read()
         assert not p.closed
     assert p.closed  # parquet file obj reports as closed
+
+
+@pytest.mark.s3
+@pytest.mark.parametrize("use_uri", (True, False))
+def test_parquet_file_with_filesystem(tempdir, s3_example_fs, use_uri):
+    s3_fs, s3_uri, s3_path = s3_example_fs
+
+    args = (s3_uri if use_uri else s3_path,)
+    kwargs = {} if use_uri else dict(filesystem=s3_fs)
+
+    table = pa.table({"a": range(10)})
+    pq.write_table(table, *args, **kwargs)
```

Comment:
```suggestion
    pq.write_table(table, path, filesystem=s3_fs)
```

(this test is about ParquetFile, so I think it is fine to always setup the test file this way

Reply:
Thanks! Would have used this suggestion directly, but it's `s3_path`, not `path`. https://github.com/apache/arrow/pull/14717/commits/4114478d79772f699ab5aa2109180f9064bace82